README.md

```diff
@@ -1,79 +1,151 @@
-   `event`, `node`, `pod`, `persistentVolume`,  are not exported at the moment.
- - You probably still need go through artifact and change hostnames and ip address.
 
+OpenShift2Nulecule creates only the following Kubernetes configurations:
+
+  - ReplicationControllers
+  - PersistentVolumeClaims
+  - Services
+
+OpenShift2Nulecule creates only the following OpenShift configurations:
+
+  - service
+  - deploymentConfig
+  - buildConfig
+  - imageStream
+  - replicationController
+  - persistentVolumeClaim
+
+  Specifically, OpenShift2Nulecule does not export these OpenShift
+  configurations:
+
+    - build
+    - imageStreamTag
+    - imageStreamImage
+    - event
+    - node
+    - pod
+    - persistentVolume
+ 
+**You advised to carefully review the created artifacts and change
+hostnames and ip addresses.**
 
-# External dependencies
-OpenShift client (`oc`) has to be installed and configured to 
-connect to OpenShift server.
+For more information about
+OpenShift object types, please see the [OpenShift
+Documentation](https://docs.openshift.com/enterprise/3.0/cli_reference/basic_cli_operations.html#object-types)
+on this topic.
 
-You can provide path to `oc` binary using `--oc` argument.
-If you need pass `--config` option to `oc` binary, you can do that using `--oc-config` argument.
+# External Dependencies
+
+The OpenShift client (`oc`) has to be installed and configured to connect
+to OpenShift server.  Additionally, the workstation running
+`openshift2nulecule` must have a working copy of the `docker` cli
+installed.
+
+You can provide the path to `oc` binary using `--oc` argument. If you
+need to pass a `--config` option to the `oc` binary, use the `--oc-config`
+option.
 
 # Usage
-Before running this you have to be authenticated to OpenShift using `oc login` command. (see [Basic Setup and Login](https://docs.openshift.com/enterprise/3.0/cli_reference/get_started_cli.html#basic-setup-and-login)).
 
-- Export whole project
+Before running OpenShift2Nulecule you have to be
+authenticated to OpenShift using the `oc login`
+command. For more information, see [Basic Setup and
+Login](https://docs.openshift.com/enterprise/3.0/cli_reference/get_started_cli.html#basic-setup-and-login).
+
+## Export A Whole Project
 
 ```sh
-openshift2nulecule --output=/path/to/new/myapp --project=myproject
+openshift2nulecule --output /path/to/new/myapp --project myproject
 ```
-This will export whole project `myproject` from OpenShift
-and create new Nulecule application in `/path/to/new/myapp` directory.
+This will export the whole OpenShift project, `myproject`, and create
+a new Nulecule application in the `/path/to/new/myapp` directory.
 
-- Selectively export an application based on labels
+## Selectively Export an Application Based on Labels
 
 ```bash
-openshift2nulecule --project hexboard --output ~/exported/hexboard --selector app=sketchpod
+openshift2nulecule --output ~/exported/hexboard --project hexboard --selector app sketchpod
```

Comment:
`=` between `app` and `sketchopd` is required. Because whole thing (`app=sketchopod`) is value for a `--selector` option. 
It should look like this:`openshift2nulecule --output ~/exported/hexboard --project hexboard --selector app=sketchpod`